src/components/datatables/Actions.tsx

```diff
@@ -14,6 +14,10 @@ import { InputField } from '../forms/InputField';
 import Select, { MultiValue, SingleValue, StylesConfig } from 'react-select';
 import { ReactNode, ChangeEvent } from 'react';
 
+interface Placeholders {
+  client_status: string;
```

Comment:
This is not following the conventions of the project. Make sure this is `clientStatus`.

Reply:
The logic of getting the placeholder string was via `query_key` and that's why we had `client_status` out of convention. There's no way if I'm right to do that with `clientStatus` because the actual query_key is client_status. So I changed the logic of this part a bit and now I think this part is conventional.